react/src/components/masthead/SprkMasthead.stories.js

```diff
@@ -268,6 +269,7 @@ export const extended = () => (
     bigNavLinks={links}
     selector={selector}
     narrowSelector={selector}
+    narrowNavMenuToggleAriaControls="sprk-masthead-narrow-nav-toggle-aria-extended"
```

Comment:
Lets just remove this, we want them to use the default. Just make sure to have a test for this.